frontend/app-development/features/overview/components/Overview.tsx

```diff
@@ -17,7 +17,7 @@ export const Overview = () => {
       <main className={classes.container}>
         {/* According to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header, the role of <header> should implicitly be "generic" when it is a descendant of <main>, but Testing Library still interprets it as "banner". */}
         <header className={classes.header} role='generic'>
-          <Header />
+          <Header /> d
```

Comment:
This change was done just to trigger Playwright

Reply:
It would be better to add compose.yaml in the paths of run-playwright-on-pr workflow.

Reply:
Agree, let me fix that!